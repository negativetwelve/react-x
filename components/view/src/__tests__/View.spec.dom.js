// Modules
import React from 'react';
import View from '../View';

/* eslint-disable no-undef */
describe('View', () => {
  context('with no props', () => {
    itRenders(() => <View />);
  });

  context('with style prop', () => {
    itRenders(() => <View style={{padding: 12}} />);
  });
});
