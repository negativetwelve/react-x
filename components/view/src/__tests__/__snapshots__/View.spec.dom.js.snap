// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`View with no props renders snapshot 1`] = `
<div
  className="css-view-1dbjc4n"
/>
`;

exports[`View with style prop renders snapshot 1`] = `
<div
  className="css-view-1dbjc4n"
  style={
    Object {
      "paddingBottom": "12px",
      "paddingLeft": "12px",
      "paddingRight": "12px",
      "paddingTop": "12px",
    }
  }
/>
`;
